### ISSUE TITLE
[Rule Request] [TD] No whitespace after TODO - "TODO     (Author)" -> "TODO(Author)"

### ISSUE THREAD

**Issue by david-fischer** [astral-sh/ruff#11323]
Hi ruff-team,

I noticed that the TD-rules do **not** get triggered on arbitrary many spaces between TODO and author:
```python
def function_with_todo() -> None:
    return  # TODO              (Test): This does not trigger any TD rule.
    # https://some.link
```

I would find it easier to search the codebase for TODOs related to a given person if I would not need to include the possibility of whitespace in the regex. Hence I would suggest to autofix this to `# TODO(Test):...`.

Thanks for all your great work!
